cmd/lncli/commands.go

```diff
@@ -1012,6 +1017,11 @@ func closeAllChannels(ctx *cli.Context) error {
 		return err
 	}
 
+	prompt := "Do you really want to close ALL channels? (yes/no): "
+	if !ctx.Bool("skip_confirmation") && !promptForConfirmation(prompt) {
+		return fmt.Errorf("action aborted by user")
```

Comment:
Is there any reason you used fmt.Errorf here instead of errors.New since nothing is being formatted?

Reply:
No specific reason, mostly just habit. But you're right, makes more sense. Changed.